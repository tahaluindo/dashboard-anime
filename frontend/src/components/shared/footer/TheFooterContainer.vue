<template>
  <article
    class="container grid grid-cols-1 mx-auto md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-x-7 gap-y-12"
  >
    <slot />
  </article>
</template> 

<script lang="ts">
import { defineComponent } from "vue";

export default defineComponent({});
</script>

<style lang="postcss" scoped>
</style>