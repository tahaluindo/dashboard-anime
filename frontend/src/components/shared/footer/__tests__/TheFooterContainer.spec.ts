import { mount } from "@vue/test-utils";
import TheFooterContainer from "../TheFooterContainer.vue";

describe("TheFooterContainer component test", () => {
  it("should render the default component", () => {
    const wrapper = mount(TheFooterContainer, {
      slots: {
        default: "<div> SLOT </div>",
      },
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
