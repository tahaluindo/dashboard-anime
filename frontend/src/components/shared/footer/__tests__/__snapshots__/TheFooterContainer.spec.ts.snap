// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheFooterContainer component test should render the default component 1`] = `
"<article class=\\"container grid grid-cols-1 mx-auto md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4 gap-x-7 gap-y-12\\">
  <div> SLOT </div>
</article>"
`;
